lib/cartopy/tests/crs/test_oblique_mercator.py

```diff
@@ -0,0 +1,193 @@
+# Copyright Cartopy Contributors
+#
+# This file is part of Cartopy and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
+"""
+Tests for the Oblique Mercator projection.
+
+"""
+
+from copy import deepcopy
+from typing import Dict, List, NamedTuple, Tuple
+
+import numpy as np
+import pytest
+
+import cartopy.crs as ccrs
+from .helpers import check_proj_params
+
+
+@pytest.fixture
+def oblique_mercator() -> ccrs.ObliqueMercator:
+    return ccrs.ObliqueMercator()
+
+
+@pytest.fixture
+def rotated_mercator() -> ccrs.ObliqueMercator:
+    return ccrs.ObliqueMercator(azimuth=90.0)
+
+
+@pytest.fixture
+def plate_carree() -> ccrs.PlateCarree:
+    return ccrs.PlateCarree()
+
+
+class TestCrsArgs:
```

Comment:
@trexfeathers Minor, now, is it `TestCRSArgs` or `TestCrsArgs` :wink:

Reply:
This was a conscious choice - I struggle to read camel case when there's an acronym in there.

Based on [this Stack Overflow thread](https://stackoverflow.com/questions/15526107/acronyms-in-camelcase) it's clearly quite controversial, but there is majority support for `TestCrsArgs`.